contribute/style-guides/redux.md

```diff
@@ -117,17 +117,48 @@ Type '{ nonExistingProperty: string; data: string[]; }' is not assignable to typ
   Object literal may only specify known properties, and 'nonExistingProperty' does not exist in type 'ExampleReducerState'. [2322]
 ```
 
-## New test functionality
+## Test functionality
 
 ### reducerTester
 
 Fluent API that simplifies the testing of reducers
 
-#### Example
+#### Simple Example
 
 ```typescript
 reducerTester()
   .givenReducer(someReducer, initialState)
   .whenActionIsDispatched(someAction('reducer tests'))
   .thenStateShouldEqual({ ...initialState, data: 'reducer tests' });
 ```
+
+#### Complex Example
+Sometimes you encounter a `resulting state` that contains properties that are hard to compare like `Dates` but you still 
```

Comment:
```suggestion
Sometimes you encounter a `resulting state` that contains properties that are hard to compare, such as `Dates`, but you still 
```
"compare like dates" sounds like you want to compare the properties _as_ Dates, which I don't think is the intention here?